content/slides/en.yaml

```diff
@@ -164,13 +270,19 @@ levels:
                 {print} Hello name
                 age {is} {ask} How old are you?
                 {print} name is age years old.
+            notes: |-
+                Point out that the `{echo}` command is no longer needed, so it will no longer work in Hedy.
+                The students can now make Rock, Paper Scissors part 2 and Haunted House.
         5:
             header: Sleep
             text: You can also use the `{sleep}` command to pause the code for a while.
             code: |-
                 {print} My favorite colour is...
                 {sleep} 2
                 {print} green!
+            notes: |-
+                Show the students the sleep command. Ask them what they think the 2 stand for.
+                Change the 2 into another number to show that they can decide how long the code will be paused.
```

Comment:
"Change the 2 into another number to show that they can decide **for how many seconds** the code will be paused"